scipy/special/tests/test_logsumexp.py

```diff
@@ -139,6 +139,11 @@ def test_logsumexp_b_shape():
 
     logsumexp(a, b=b)
 
+def test_logsumexp_precision():
+    a = [0, -40]
+    expected = 4.248354255291589e-18
```

Comment:
Please add a comment on how to reproduce this value. mpmath or Wolfram Mathematica incantation or how whatever other way you used to reproduce this.